src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,35 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+
+        //Excavator excavator = new Excavator();
+        //Bulldozer bulldozer = new Bulldozer();
+        //Truck truck = new Truck();
+
+        Object[] object = new Object[3];
+
+        object[0] = new Excavator();
+        object[1] = new Bulldozer();
+        object[2] = new Truck();
```

Comment:
You should use abstract references instead of specific ones. This allows you to take advantage of polymorphism. For instance, use 'Machine[] machines = new Machine[3];' instead of 'Object[] object = new Object[3];'.